openapi/openapi.yaml

```diff
@@ -399,12 +539,44 @@ components:
       required:
         - count
 
+    SemanticVersion:
+      description: 'A version string in the Semantic Version form'
+      type: string
+      pattern: ^((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)$
+      example: '1.0.1'
+
     User:
       title: 'User'
       description: 'Automation Hub User'
       type: object
 
   parameters:
+
+    CollectionNamespaceName:
```

Comment:
believe this and NamespaceName can be combined

Reply:
Please do

Reply:
Looking closer, I can't[1] combine them because they represent different path param.
CollectionNamespaceName is the {namespace}  used in `/collections/{namespace}/{name}`.
And NamespaceName is the {name} used in `/namespaces/{name}`

[1] Can't combine them without changing the path param names that is.

But while CollectionNamespaceName is used in 5 places, NamespaceName is only used in one so it can be inlined.

Reply:
updated